cmd/tracking/main.go

```diff
@@ -0,0 +1,105 @@
+// What it does:
+//
+// This example uses one of the Tracker classes from opencv_contrib to track a region of interest (e.g. a face)
+// and draws a rectangle around it, before displaying it within a Window.
+//
+// in this example, users have to select an initial roi with the mouse, and press enter, to start the tracking
+// (but the roi could also come from e.g. a previous cascade based detection)
+//
+// also see https://docs.opencv.org/master/d2/d0a/tutorial_introduction_to_tracker.html for a tutorial
+//
+// How to run:
+//
+// tracking [camera ID]
+//
+// 		go run ./cmd/tracking/main.go 0
+//
+// +build example
+
+package main
+
+import (
+	"fmt"
+	"image/color"
+	"os"
+	"strconv"
+
+	"gocv.io/x/gocv"
+	"gocv.io/x/gocv/contrib"
+)
+
+func main() {
+	if len(os.Args) < 2 {
+		fmt.Println("How to run:\n\ttracking [camera ID]")
+		return
+	}
+
+	// parse args
+	deviceID, _ := strconv.Atoi(os.Args[1])
+
+	// open webcam
+	webcam, err := gocv.VideoCaptureDevice(int(deviceID))
+	if err != nil {
+		fmt.Printf("error opening video capture device: %v\n", deviceID)
+		return
+	}
+	defer webcam.Close()
+
+	// open display window
+	window := gocv.NewWindow("Tracking")
+	defer window.Close()
+
+	// create a tracker instance
+	// (one of Mil,Kcf,Tld,MedianFlow,Boosting or Mosse)
+	tracker := contrib.NewTrackerMosse()
```

Comment:
so I think you meant `NewTrackerMOSSE` now

Reply:
harrr....